docs/source/_src_user_guide/state_propagation/environment_setup/create_bodies/available.rst

```diff
@@ -0,0 +1,129 @@
+  
+* `Gravity field models <https://py.api.tudat.space/en/latest/gravity_field.html>`_, to be assigned to the :attr:`~tudatpy.numerical_simulation.environment_setup.BodySettings.gravity_field_settings` attribute of :class:`~tudatpy.numerical_simulation.environment_setup.BodySettings`.  
+
+  * These models provide various ways in which to define the gravitational field of solar system bodies. Note: the mass associated with these gravitational field is the gravitational mass, which does *not* need to be equal to its inertial mass.
+  
+* `Gravity field variation models <https://py.api.tudat.space/en/latest/gravity_field_variation.html>`_, to be assigned to the :attr:`~tudatpy.numerical_simulation.environment_setup.BodySettings.gravity_field_variation_settings` attribute of :class:`~tudatpy.numerical_simulation.environment_setup.BodySettings`. Note: this attribute is a list, and any number of variation models may be added.  
+
+  * These models provide various ways in which to define the time-variability of a body's (spherical harmonic) gravitaty field.
+  
+* `Rotation models <https://py.api.tudat.space/en/latest/rotation_model.html>`_, to be assigned to the :attr:`~tudatpy.numerical_simulation.environment_setup.BodySettings.rotation_model_settings` attribute of :class:`~tudatpy.numerical_simulation.environment_setup.BodySettings`. 
+
+  * These models provide various ways in which to define the orientation of a body w.r.t. inertial space, and produces a quaternion/rotation matrix, and angular velocity vector/rotation matrix derivative. Note that Tudat can also produce such models by numerical propagation of the Euler equations (see :ref:`rotational_dynamics`).
+  
+* `Shape models <https://py.api.tudat.space/en/latest/shape.html>`_, to be assigned to the :attr:`~tudatpy.numerical_simulation.environment_setup.BodySettings.shape_settings` attribute of :class:`~tudatpy.numerical_simulation.environment_setup.BodySettings`. 
+
+  * These models provide various ways in which to define the exterior of a *natural* body and is typically used to calculate (for instance) altitude, ground station position, etc. Note: the exterior shape of an artificial body, from which aerodynamic and radiation pressure properties can be evaluated, uses a different interface, which is currently under development
+
+* `Radiation pressure <https://py.api.tudat.space/en/latest/radiation_pressure.html>`_, to be assigned to the :attr:`~tudatpy.numerical_simulation.environment_setup.BodySettings.rotation_model_settings` attribute of :class:`~tudatpy.numerical_simulation.environment_setup.BodySettings`. Note: this attribute is a dictionary, with one radiation pressure model per source body. 
+
+  * These models provide various ways in which to define the response of a body to incident radation pressure.
+
+.. _specific_environment_considerations:
+
+
+Specific Considerations
+=======================
+
```

Comment:
Top level comments:

- I find this section is extremely useful to highlight some important (environment) model choices that are sort of specific to Tudat. As mentioned during last meeting, I think that it would be nice to add a link to this on the main environment page (I know that I already mentioned that, just adding it here as a reminder :) ).
- Overall, I am not entirely convinced by the subsections order. I would suggest to either follow the alphebetical order, or agree on an "order of importance" for the different environment models that we could stick to on **all** environment-related pages. I noticed that the order in which models are discussed is for instance different from the one used in "Interacting with the environment during propagation".
- Concerning the title, I think that we discussed changing it to "Points of attention" during the meeting(?), but I don't know what are your thoughts on this?
- My apologies in advance if the following is only a matter of personal taste/interpretation: If I understand correctly the idea of this page is to highlight some points of attention in order to set up appropriate environment models, and not to discuss all models (dedicated page for this). This is in practice exactly what it does but I have the feeling that it might not always be entirely clear why some models are mentioned here (i.e. what makes them different and therefore worth an additional piece of "warning"). This information is actually there, but may somewhat not be immediately apparent to the new Tudat user. Maybe just reformulating/restructuring the introduction paragraph of each subsection would be enough? What I have in mind is a short sentence highlight which models will be elaborated upon and why. I'm not sure if the above is clear, and maybe this feeling also is not shared by others(?). My apologies if this comment ends up being irrelevant!

Reply:
You don't think "whichever order it popped into my head" is a good paradigm for deciding the order? I'm shocked! Shocked and insulted, I say! ;)

Your last point is a good one. How about:
* Adding a link to the specific subsection of points of attention here: https://github.com/tudat-team/tudat-space/blob/feature/environment_revision/docs/source/_src_user_guide/state_propagation/environment_setup/create_bodies/available.rst#available-model-types (per relevant model)
* Adding a table of contents with a one line explanation per model here: https://github.com/tudat-team/tudat-space/blob/feature/environment_revision/docs/source/_src_user_guide/state_propagation/environment_setup/create_bodies/available.rst#specific-considerations